pax-core/src/rendering.rs

```diff
@@ -209,22 +208,34 @@ pub trait InstanceNode {
         expanded_node: Option<&ExpandedNode>,
     ) -> std::fmt::Result;
 
-    /// Expands the current `InstanceNode` into a stateful `ExpandedNode`, with its own instances of properties & common properties, in the context of the
-    /// provided `PropertiesTreeContext`.  Node expansion takes into account the "parallel selves" that an `InstanceNode` may have through the
-    /// lens of declarative control flow, [`ConditionalInstance`] and [`RepeatInstance`].
+    // /// Used by elements that need to communicate across native rendering bridge (for example: Text, Clipping masks, scroll containers)
+    // /// Called by engine after [`expand_node`], passed calculated size and transform matrix coefficients for convenience
+    // /// Expected to induce side-effects (if appropriate) via enqueueing messages to the native message queue
+    // ///
+    // /// An implementor of `handle_native_patches` is responsible for determining which properties if any have changed
+    // /// (e.g. by keeping a local patch object as a cache of last known values.)
```

Comment:
Can these be plain `///` rustdoc comments, or were the `//`s added to hack around a cargo error?

Reply:
Oops! Not intended.